tlsn/examples/twitter/twitter_dm.rs

```diff
@@ -147,26 +146,28 @@ async fn main() {
     let mut prover = prover.start_notarize();
 
     // Identify the ranges in the transcript that contain secrets
-    let (public_ranges, private_ranges) = find_ranges(
+    let (public_ranges, _private_ranges) = find_ranges(
         prover.sent_transcript().data(),
         &[
             access_token.as_bytes(),
             auth_token.as_bytes(),
             csrf_token.as_bytes(),
+            client_uuid.as_bytes(),
         ],
     );
 
     let recv_len = prover.recv_transcript().data().len();
 
     let builder = prover.commitment_builder();
 
-    // Commit to the outbound transcript, isolating the data that contain secrets
-    for range in public_ranges.iter().chain(private_ranges.iter()) {
-        builder.commit_sent(range.clone()).unwrap();
-    }
-
-    // Commit to the full received transcript in one shot, as we don't need to redact anything
-    builder.commit_recv(0..recv_len).unwrap();
+    // Collect commitment ids for the outbound transcript
+    // Commit to everything but the redacted tokens
+    let mut commitment_ids = public_ranges
+        .iter()
+        // .chain(private_ranges.iter())
```

Comment:
is this commented code needed?

Reply:
@th4s I extended the example to also generate a proof.
I think it is more clear to only commit to public data. This avoids the "magic numbers" when revealing commits (like in https://github.com/tlsnotary/tlsn/blob/9169088258751976ec0e499392dceb116021c92a/tlsn/examples/discord/discord_dm.rs#L177 <= "Why is "2" skipped and not "1"?)